src/server_manager/web_app/digitalocean_account.ts

```diff
@@ -23,12 +23,40 @@ import * as do_install_script from "../install_scripts/do_install_script";
 const SHADOWBOX_TAG = 'shadowbox';
 const MACHINE_SIZE = 's-1vcpu-1gb';
 
+export interface ShadowboxSettings {
+  imageId: string;
+  metricsUrl: string;
+  sentryApiUrl?: string;
+  watchtowerRefreshSeconds?: number;
+}
+
+export enum Status {
+  ACTIVE,
+  EMAIL_UNVERIFIED,
+  INVALID_BILLING_INFORMATION,
+}
+
 export class DigitalOceanAccount implements Account {
   private servers: DigitalOceanServer[] = [];
 
   constructor(
-      private digitalOcean: DigitalOceanSession, private image: string, private metricsUrl: string,
-      private sentryApiUrl: string|undefined, private debugMode: boolean) {}
+      private digitalOcean: DigitalOceanSession, private shadowboxSettings: ShadowboxSettings,
```

Comment:
Perhaps we should pass email (and later id) in the constructor instead.
The id should never change and we never really update the email.

Reply:
Can we do this in a followup? I want to give it some more thought first.